tests/test_debug/test_snapshot.py

```diff
@@ -642,3 +644,93 @@ def test_snapshot_save_stdout():
         expected_svg = expected_svg_file.read()
 
     assert result.stdout == expected_svg
+
+
+def func_with_include(include: Optional[tuple[str, ...]] = None):
```

Comment:
See above note about type annotation of `include` parameter. Also add a return type annotation.

(Applies to all helper functions defined here.)